pkg/deployment/deployment.go

```diff
@@ -315,6 +315,32 @@ func (builder *Builder) WithServiceAccountName(serviceAccountName string) *Build
 	return builder
 }
 
+// WithVolume sets list of volumes that can be mounted by containers belonging to the deployment.
```

Comment:
Please update the comment to reflect that the updated method only adds one volume instead of a list

Reply:
`WithVolume attaches given volume to a deployment`

Reply:
thanks, updated